community/security/src/main/java/org/neo4j/server/security/auth/BasicAuthManagerFactory.java

```diff
@@ -34,6 +38,23 @@
 @Service.Implementation( AuthManager.Factory.class )
 public class BasicAuthManagerFactory extends AuthManager.Factory
 {
+    private static final String USER_STORE_FILENAME = "auth";
+
+    public static Path getUserStoreFile( Config config )
```

Comment:
This can return a `UserRepository` directly, indeed the returned `Path` is always used to construct a `FileUserRepository`